mlflow/protos/service.proto

```diff
@@ -625,6 +625,9 @@ message RunInfo {
   // be removed in a future MLflow version.
   optional string run_uuid = 1;
 
+  // The name of the run.
+  optional string run_name = 3;
```

Comment:
Ditto with https://github.com/mlflow/mlflow/pull/6769/files#r969066361. Can we use 15 and check git history to make sure it's safe? We really shouldn't have ever deleted fields from our protos...

Reply:
https://github.com/mlflow/mlflow/pull/6769#discussion_r969068536